src/ApiBuilder.php

```diff
@@ -116,6 +116,7 @@ public function __call($name, $arguments)
 
             // Method
             $method = strtoupper(array_get($api, 'method', 'GET'));
+            $request_type = array_get($api, 'request_type');
```

Comment:
use camelcase in variable name

Reply:
Changes the variable name with camlecase